docs/user-guide.md

```diff
@@ -362,6 +369,42 @@ spec:
 Note that the mutating admission webhook is needed to use this feature. Please refer to the 
 [Quick Start Guide](quick-start-guide.md) on how to enable the mutating admission webhook.
 
+### Python Support
+
+Python support can be enabled by setting `mainApplicationFile` with path to your python application.
+Optionaly, `pythonVersion` parameter can be used to set the major Python version of the docker image used 
+to run the driver and executor containers. Below is an example showing part of a `SparkApplication` specification:
+
+```yaml
+spec:
+  type: Python
+  pythonVersion: 2
+  mainApplicationFile: local:///opt/spark/examples/src/main/python/pyfiles.py
+```
+
+Some PySpark applications need additional Python packages additionally to the main application resource to run. 
+Such dependencies are specified using the `--py-files` option of `spark-submit` command. 
```

Comment:
I would rephrase this sentence as "Such dependencies are specified using the optional field `.spec.deps.pyFiles` , which translates to the `--py-files` option of the `spark-submit` command.".